css/main.css

```diff
@@ -325,3 +326,78 @@ li {
 .skills {
   list-style: inside url("../media/svg/circle.svg");
 }
+
+.flex-prop {
+  display: flex;
+  align-items: center;
+}
+
+.contact-container {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
```

Comment:
**Required**
- [ ] As mentioned by the last reviewer, though you might have created a `flex-prop` rule, the `display: flex` rule is still being duplicated on lines 331 & 336. I think this can still be improved.